src/prpy/tsr/tsr.py

```diff
@@ -41,6 +48,8 @@ def __init__(self, T0_w=None, Tw_e=None, Bw=None,
             Tw_e = numpy.eye(4)
         if Bw is None:
             Bw = numpy.zeros((6, 2))
+        elif any((Bw[:, 1]-Bw[:, 0]) > 2*pi + EPSILON):
+            raise(ValueError('Bw range must be within 2*PI'))
```

Comment:
Two quick questions about "what it means to be a TSR":

1. Must all W transforms contain the identity pose (i.e. must the first column of Bw be all-nonpositive, and the second be all-nonnegative)?  There's no fundamental reason why this must be so, but IIRC CBiRRT does not like bounds on W that exclude identity.

2. Dmitry's TSR code [0] claims that Bw[:,0] can be greater than Bw[:,1], signifying "an axis flip (not an error)".  Does this code support this?  If so, this check should have an absolute value in it.

In any case, it might be a good idea to write up a small section on the prpy README describing what a TSR is.

[0] https://github.com/personalrobotics/comps/blob/master/cbirrt2/TaskSpaceRegion.cpp#L65